scene/resources/atlas_texture.cpp

```diff
@@ -132,12 +144,16 @@ void AtlasTexture::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_margin", "margin"), &AtlasTexture::set_margin);
 	ClassDB::bind_method(D_METHOD("get_margin"), &AtlasTexture::get_margin);
 
+	ClassDB::bind_method(D_METHOD("set_anchor", "anchor"), &AtlasTexture::set_anchor);
+	ClassDB::bind_method(D_METHOD("get_anchor"), &AtlasTexture::get_anchor);
```

Comment:
This shouldn't be bound here as it's already bound in the parent, that's probably what's causing the errors
```suggestion
```

Reply:
Ok, thanks, I'm not really familiar yet with all this binding thing in Godot.

Reply:
It can be confusing indeed